Workaround for qprinter.h not being found when the Qt stack has been built
without CUPS support.

/usr/local/lib/python3.9/site-packages/PyQt6/bindings/QtPrintSupport/qprinter.sip:28:10: fatal error: 'qprinter.h' file not found
#include <qprinter.h>
         ^~~~~~~~~~~~

--- pyproject.toml.orig	2023-10-25 10:26:54 UTC
+++ pyproject.toml
@@ -32,3 +32,5 @@ qmake-QT = ["webenginewidgets"]
 
 [tool.sip.bindings.QtWebEngineWidgets]
 qmake-QT = ["webenginewidgets"]
+define-macros = ["QT_PRINTSUPPORT_LIB"]
+include-dirs = ["%%QT_INCDIR%%/QtPrintSupport"]
